react-frontend/src/components/StyleComponents/pageContent.js

```diff
@@ -8,6 +8,38 @@ export const GlobaStyleSize = createGlobalStyle`
  }
 `;
 
+export const SimplePageBody = styled.div.attrs({
+  id: 'content-body',
+})`
+  margin: auto;
+  @media only screen and (max-width: 800px) {
+    width: 85%;
+  }
+`;
+
+//TODO: When Product page and Digital Framework page refactored
+//delete these and used the common 'SimplePageBody' component
+
+export const SimplePageBodyDigital = styled.div.attrs({
+  id: 'digital-content-body',
+})`
+  width: 960px;
+  margin: auto;
+  @media only screen and (max-width: 800px) {
+    width: 85%;
```

Comment:
Could you replace lines 26-29, the styles for `SimplePageBodyDigital` with the following:

```
  margin: auto;
  padding: 0 15px;
  width: 100%;
  @media only screen and (min-width: 576px) {
    max-width: 540px;
  }
  @media only screen and (min-width: 768px) {
    max-width: 720px;
  }
  @media only screen and (min-width: 992px) {
    max-width: 960px;
  }
  @media only screen and (min-width: 1200px) {
    max-width: 1140px;
  }

```

This would be the "global container" that I'd like to be able to implement within all main blocks to achieve alignment.

Reply:
This part is outside the scope of the ticket.  I'm fixing the other page bodies as I go, the reason for creating the digital and product was so I could fix them page by page without the PR being 80 files.  Getting rid of those two should be tackled separately.